versions.tf

```diff
@@ -1,10 +1,21 @@
 terraform {
-  required_version = ">= 0.12.0"
-
+  required_version = ">= 0.12"
   required_providers {
-    aws      = ">= 2.0"
-    template = "~> 2.0"
-    local    = "~> 1.2"
-    null     = "~> 2.0"
+    aws = {
```

Comment:
Doesn't this new format require Terraform version to be >= 0.13?

Reply:
Yes the below is a terraform 0.13 thing, not aws provider 3.x